frontend/src/components/Logout.jsx

```diff
@@ -3,13 +3,13 @@ import BCButton from '@/components/BCButton'
 import { logout } from '@/utils/keycloak'
 import { useCurrentUser } from '@/hooks/useCurrentUser'
 
-const Logout = () => {
+export const Logout = () => {
```

Comment:
Is removing the default just a stylistic choice?

Reply:
moves it inline with our other components that follow explicit export/imports